src/tracks/playabletrack/wavetrack/ui/WaveChannelView.cpp

```diff
@@ -1705,7 +1705,10 @@ std::weak_ptr<WaveClip> WaveChannelView::GetSelectedClip()
 {
    if (auto affordance = std::dynamic_pointer_cast<WaveTrackAffordanceControls>(GetAffordanceControls()))
    {
-      return affordance->GetSelectedClip();
+      if(auto interval = *affordance->GetSelectedInterval())
+      {
+         return interval->GetClip(GetChannelIndex());
```

Comment:
I want only constant `0` as an argument of `GetClip` and later I will remove the argument.

But the outer `if` should justify `assert(GetChannelIndex() == 0);`

So write that assertion, then change the argument of `GetClip`.

Actually `GetChannelIndex()` still always returns 0, unlike `ReallyGetChannelIndex`

The problem of unifying those two functions (keeping the name of the first, and the behavior of the second) is another box on my checklist for later.